content/reference/_index.md

```diff
@@ -4,3 +4,25 @@ weight: 10000
 icon: "lightbulb"
 description: "Reference material for Upbound"
 ---
+
+## Command-Line Interface (CLI)
```

Comment:
If the contents are going to be just a listing of the files should we make this dynamic? 

We can do something like this to iterate over the child pages for the `_index.md` pages.

```html
    {{if .Page.IsSection }}
        <ul>
        {{ range .Pages }}
            <li><a href="{{.Permalink}}">{{.Title}}</a></li>
        {{ end }}
        </ul>
    {{ end }}
```

inside `themes/upbound/layouts/partials/body/content-body.html`

Reply:
Agreed, I'll update that.